components/zoom_admin/sources/meeting-started/meeting-started.mjs

```diff
@@ -19,11 +20,12 @@ export default {
   },
   async run(event) {
     const { payload } = event;
-    const { object } = payload;
     this.$emit(event, {
-      summary: `Meeting ${object.topic} started`,
-      id: object.uuid,
-      ts: +new Date(object.start_time),
+      summary: `Meeting ${payload?.object?.topic} started`,
+      id: payload?.object?.uuid ?? uuidv4(),
+      ts: payload?.object?.start_time
+        ? +new Date(payload.object.start_time)
+        : new Date(),
```

Comment:
Checking for `payload?` is not necessary, since it is being destructured from `event`.

```suggestion
      summary: payload.object?.topic
        ? `Meeting ${payload.object.topic} started`
        : `Meeting started`,
      id: payload.object?.uuid ?? uuidv4(),
      ts: payload.object?.start_time
        ? +new Date(payload.object.start_time)
        : new Date(),
```

Reply:
That's the reason for the bug :)
`event.payload` may be `null` in some cases.

Reply:
Oh right, I though it was `object` that was `undefined`, my bad

Reply:
Np sir!

Reply:
Indeed :)